test/eredis_sentinel_SUITE.erl

```diff
@@ -143,6 +144,35 @@ t_reconnect_success_on_sentinel_connection_break(Config) when is_list(Config) ->
     process_flag(trap_exit, false),
     ?assertEqual(died, IsDead).
 
+t_reconnect_success_on_sentinel_connection_break_mix_endpoints(Config) when is_list(Config) ->
```

Comment:
Please add some comments inside this test case to make it easier to understand what's happening.